Model/Hexed.cs

```diff
@@ -56,17 +62,19 @@ public class Client
 
         private static int MAX_POSSIBLE_HP = 1000000;
         private string execName { get; set; }
+        public string processName { get; private set; }
         private Utils.ProcessMemoryReader PMR { get; set; }
         private int currentNameAddress { get; set; }
         private int currentHPBaseAddress { get; set; }
         private int statusBufferAddress { get; set; }
         private int _num = 0;
 
-        public Client(string execName, int currentHPBaseAddress, int currentNameAddress)
+        public Client(string execName, int currentHPBaseAddress, int currentNameAddress, string processName)
```

Comment:
You don't need this extra property `processName` . Exec name is exactly you want :D